Workbooks/Performance/Apdex/Apdex.workbook

```diff
@@ -191,85 +151,184 @@
           {
             "id": "2f57173e-c1de-4b51-90a6-fc4baee9fec6",
             "version": "KqlParameterItem/1.0",
-            "name": "SegmentBy",
+            "name": "SplitBy",
```

Comment:
does users blade pass this in when you open apdex from there?  if so, it won't be hooked up anymore.